acra/guides/trying-acra-with-docker/_index.md

```diff
@@ -0,0 +1,403 @@
+
+Please set `ACRA_MASTER_KEY` environmental variable, or check [KeyManagement]({{< ref "acra/acra-in-depth/cryptography-and-key-management/#-INVALID" >}}) page for more details:
+```bash
+
+export ACRA_SERVER_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
+export ACRA_CONNECTOR_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
+```
+
+Define the Client ID:
+```bash
+export ACRA_CLIENT_ID="MyClientID"
+```
+
+Optionally you may specify a docker image tag, which can be one of the following:
+
+* `stable` or `latest` - stable branch, recommended, default;
+* `master` or `current` - master branch of the GitHub repository;
+* `<full_commit_tag>` - specify the exact commit in repository;
+* `<version>` - choose version tag.
+
+```bash
+# Examples:
+# branch
+export ACRA_DOCKER_IMAGE_TAG="master"
+# commit tag
+export ACRA_DOCKER_IMAGE_TAG="2d2348f440aa0c20b20cd23c49dd34eb0d42d6a5"
+# version
+export ACRA_DOCKER_IMAGE_TAG="0.76-33-g8b16bc2"
+```
+
+Please define the database name and user credentials:
+
+```bash
+# for Postgresql
+export POSTGRES_DB="<db_name>"
+export POSTGRES_USER="<user_name>"
+export POSTGRES_PASSWORD="<user_password>"
+
+# for MySQL
+export MYSQL_ONETIME_PASSWORD="<mysql_onetime_password>"
+export MYSQL_ROOT_PASSWORD="<mysql_root_password>"
+export MYSQL_DATABASE="<db_name>"
+export MYSQL_USER="<user_name>"
+export MYSQL_PASSWORD="<user_password>"
+```
+
+In order to access AcraWebConfig HTTP interface, you can define:
+```bash
+export ACRA_HTTPAUTH_USER=<http_auth_user>
+export ACRA_HTTPAUTH_PASSWORD=<http_auth_password>
+```
+
+Now you can run `docker-compose`:
+```bash
+    docker-compose -f docker/<compose_file_name> up
+```
+
+And connect to ports described above.
+
+## Example application
+
+In the following examples, we assume that the schemes are running and all the environment variables are set as described in the code block below. For your convenience, we recommend that you perform the following actions in two different shell windows:  run the Docker in one and the examples in another.
+
+Please define the same set of environment variables in **both** shell windows before proceeding to the next steps:
+```bash
+export ACRA_SERVER_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
+export ACRA_CONNECTOR_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
+export ACRA_CLIENT_ID="test"
+export POSTGRES_DB="acra"
+export POSTGRES_USER="dbuser"
+export POSTGRES_PASSWORD="dbpassword"
+export ACRA_CLIENT_ID="acraclient"
+```    
+
+### Install the dependencies for the example application
+(shell window #2)
+
+> Important: You must start with installing Themis encryption library for Acra to work. Please follow either [these instructions]({{< ref "themis/installation/" >}}) or use a simple [script](https://pkgs.cossacklabs.com/scripts/libthemis_install.sh) to install Themis before proceeding.
```

Comment:
Hugo has its own blocks which would look better here:

```markdown
{{< hint warning >}}
**Important:**
You need to have Themis Core library installed [...]
{{< /hint >}}
```

But hey. This is stupid. You provide all these Docker Compose files for running Acra, why there isn't a premade image for running examples with all necessary dependencies already installed? That would allow users to try Acra out even on Windows. They would not need to install any shady Themis libraries just to try things out, and the instructions would be just `docker exec -it acra-examples-python /bin/bash` to dive into a container that has everything ready to go.